examples/hybrid-compute/Quantinuum_hseries_hybrid_compute.ipynb

```diff
@@ -0,0 +1,1208 @@
+    "    * For example, examine the call `x = wasm_func(a, b, c);` The `wasm_func` function can exist in QASM where `a`, `b`, `c`, and `x` are classical registers. \n",
+    "    * Such function calls have timing restrictions. Wasm calls in quantum programs have been run that took multiple milliseconds.\n",
+    "1. The state of a Wasm program persists between calls in the quantum programs within the limits of a job chunk window (~300 shots). \n",
+    "    * This means global variables can hold a mutable state that is modified between calls.\n",
+    "    * One can imagine using various global arrays. It's possible to have global pointers work with heap allocated data structures (for example using unsafe Rust features when compiling to Wasm).\n",
+    "1. Currently, the combined Quantum Web Assembly programs have to come in under 6 MB*."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Basic Usage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To inject Wasm into a TKET program and enable processing on the H-Series devices or emulators, three components must be incorporated in the workflow.\n",
+    "\n",
+    "1. [pytket.wasm.WasmFileHandler](https://tket.quantinuum.com/api-docs/wasm.html#pytket.wasm.WasmFileHandler): Enables addition of the Wasm module to the TKET program as well as provides capability to verify logic within the Wasm module.\n",
+    "1. [pytket.circuit.Circuit.add_wasm_to_reg](https://tket.quantinuum.com/api-docs/circuit_class.html#pytket.circuit.Circuit.add_wasm_to_reg): Adds the linked Wasm to the TKET program as a classical operation.\n",
+    "1. [wasm_file_handler](https://tket.quantinuum.com/extensions/pytket-quantinuum/api.html#pytket.extensions.quantinuum.QuantinuumBackend.process_circuits) kwarg on `QuantinuumBackend.process_circuits`: The linked Wasm module must be specified during job submission to H-Series."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Components 1 and 2 are used in the `Job Construction` subsection. Component 3 is used in the `Job Submission` subsection. \n",
+    "\n",
+    "Usage of Wasm is showcased in a Repeat Until Success (RUS) example. The RUS protocol, initially introduced in [arxiv.1311.1074](https://arxiv.org/abs/1311.1074), is a non-deterministic decompostion technique to approximate a single-qubit unitary within an allowable error threshold. The RUS circuit is composed of multiple subcircuts and each subcircuit runs successively if a specific condition is satisfied."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Circuit Construction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We illustrate an example calling Wasm from within a pytket `Circuit`.\n",
+    "\n",
+    "A 2-qubit circuit is initialized with 4 Bit registers. \n",
```

Comment:
The circuit has by default no registers. They are created when the users ask from them to be added. If the name of the registers there are default names for the first register that will be added. They will be named `c` for the classical register and `q` for the quantum reg. It is possible to create a two qubit without any classical registers and without any classical bits.

Reply:
The circuit is intentionally created with no qubits and no bits, and intentionally the qubit and bit registers are added explicitly. What are you proposing? What is said above is difficult to follow.

Reply:
specify the sentence in a way to make clear the 4 bit registers that will be manually added.